common/buildcraft/robotics/tile/TileZonePlanner.java

```diff
@@ -0,0 +1,18 @@
+package buildcraft.robotics.tile;
+
+import buildcraft.lib.tile.TileBCInventory_Neptune;
+import buildcraft.lib.tile.item.ItemHandlerManager;
+import buildcraft.lib.tile.item.ItemHandlerSimple;
+import buildcraft.robotics.ZonePlan;
+
+public class TileZonePlanner extends TileBCInventory_Neptune {
+    public final ItemHandlerSimple invPaintbrushes;
```

Comment:
Why do paintbrushes need their own inventory? Can you just manually render them all out in the gui?